lib/datasource/docker/index.js

```diff
@@ -388,6 +388,17 @@ async function getLabels(registry, repository, tag) {
       const url = `${registry}/v2/${repository}/blobs/${configDigest}`;
       const configResponse = await got(url, {
         headers,
+        hooks: {
+          beforeRedirect: [
+            options => {
+              if (options.search.indexOf('X-Amz-Algorithm') !== -1) {
```

Comment:
Can we add a defensive check that `option.search` exists, unless it's documented somewhere to be guaranteed? Don't want us blowing up some day because it stops being present.

Reply:
Good idea, I'll add that in.